src/adapters/refract/transformSections.coffee

```diff
@@ -2,37 +2,65 @@ _ = require('./helper')
 
 blueprintApi = require('../../blueprint-api')
 getDescription = require('./getDescription')
+
 transformResources = require('./transformResources')
+transformResource = require('./transformResource')
+
 
-module.exports = (element) ->
+module.exports = (parentElement) ->
   resourceGroups = []
-  resourceGroupsElements = _.chain(element)
-                    .content()
-                    .filter({element: 'category', meta: {classes: ['resourceGroup']}})
-                    .value()
-
-  # If we can't find any resource group we can assume that there are Resources directly
-  if resourceGroupsElements.length is 0
-    resourceGroups.push(
-      new blueprintApi.Section({
-        name: ''
+
+  # List of Application AST Resources (= already transformed
+  # Refract Resources into the Application AST Resource).
+  applicationAstResources = []
+
+  _.forEach(parentElement.content, (element, index) ->
```

Comment:
@smizell 

Thanks! Fixed in https://github.com/apiaryio/metamorphoses/commit/dcb2e4f1a18e8b4ee7f6f964c39f9a4d0e528573.